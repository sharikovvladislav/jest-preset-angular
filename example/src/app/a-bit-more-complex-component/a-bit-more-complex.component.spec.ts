/* tslint:disable:no-unused-variable */
import { async, ComponentFixture, TestBed } from '@angular/core/testing';

import { ABitMoreComplexComponent } from './a-bit-more-complex.component';
import { ChildComponent } from './child.component';

describe('ABitMoreComplexComponent', () => {
  let component: ABitMoreComplexComponent;
  let fixture: ComponentFixture<ABitMoreComplexComponent>;

  beforeEach(
    async(() => {
      TestBed.configureTestingModule({
        declarations: [ABitMoreComplexComponent, ChildComponent],
      })
        .overrideComponent(ChildComponent, {set: {template: 'stubbedBody'}})
        .compileComponents();
    }),
  );

  beforeEach(() => {
    fixture = TestBed.createComponent(ABitMoreComplexComponent);
    component = fixture.componentInstance;
    fixture.detectChanges();
  });

  it('should create', () => {
    expect(component).toBeTruthy();
  });

  it('snapshot test', () => {
    expect(fixture).toMatchSnapshot();
  });
});
