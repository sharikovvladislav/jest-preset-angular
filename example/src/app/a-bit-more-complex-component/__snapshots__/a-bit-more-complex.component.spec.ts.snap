// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ABitMoreComplexComponent snapshot test 1`] = `
<app-a-bit-more-complex
  anotherVar="false"
  someVar={[Function Boolean]}
>
  
      <div
    class="kek"
  >
    
        diary works!
        
    
    <div
      class="inner-class"
    >
      
          
      
      
          
      <div
      >
        another case
      </div>
      
        
    </div>
    
        
    <div
    >
      one more case case
    </div>
    
        
    <div
    >
      <app-child-component
        kek="someVar"
      >
        stubbedBody
      </app-child-component>
    </div>
    
      
  </div>
    
</app-a-bit-more-complex>
`;
